app/_locales/en/messages.json

```diff
@@ -2571,6 +2554,14 @@
     "message": "Ledger and Firefox Users Experiencing Connection Issues",
     "description": "Title for a notification in the 'See What's New' popup. Tells users that latest firefox users using U2F may experience connection issues."
   },
+  "notifications21Description": {
+    "message": "U2F and Ledger Live connection methods on chrome are now deprecated and can no longer be used. Webhid is now the only method for Ledger devices on chrome.",
+    "description": "Description of a notification in the 'See What's New' popup. Describes the U2F and Ledger Live connection modes are now deprecated"
```

Comment:
we default to webhid if the `window.navigator.hid` is available. https://github.com/MetaMask/metamask-extension/blob/ef01dc1391612aee26f2cc8bf134c85154864442/app/scripts/controllers/preferences.js#L64

For firefox it would always be U2F and for chrome now it will always be webhid.

Reply:
Revised content here:

**U2F and Ledger Live deprecation** 
U2F and Ledger Live are no longer available on Chrome. You can still connect Ledger devices on Chrome using Webhid.

Reply:
sure, i just updated the content